contrib/vcl/source/vcl_interface.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include "envoy/network/socket.h"
+
+#include "source/common/network/socket_interface.h"
+
+#include "vppcom.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Network {
+namespace Vcl {
+
+#define VCL_DEBUG (0)
+#define VCL_RX_ZC (0)
+
+#if VCL_DEBUG > 0
+#define VCL_LOG(fmt, _args...) ENVOY_LOG_MISC(debug, "[{}] " fmt, vppcom_worker_index(), ##_args)
```

Comment:
I think it should be fine to not use the ifdef here. debug and trace especially are already noisy.